main.js

```diff
@@ -256,9 +285,20 @@ $cartSummaryView.addEventListener('click', (event) => {
     app.view = 'catalog'
     renderApp(app)
   }
+  else if (event.target.id === 'cart-checkout') {
+    app.view = 'checkout'
+    renderApp(app)
+  }
 })
 
 $cart.addEventListener('click', (event) => {
   app.view = 'cart'
+  app.cart.total = calculateCartTotal(app.cart)
```

Comment:
Do you really need to calculate this here? If you know the items in the cart, you should be able to calculate it on-the-fly when your view renders.